addons/web/static/src/views/list/list_renderer.scss

```diff
@@ -78,11 +78,24 @@
 
                 display: table-cell;
                 @include text-truncate();
-                vertical-align: middle;
 
                 &.o_list_text {
                     white-space: pre-wrap;
                 }
+
+                // TODO ugly temporary overrides to visually align the field widget
+                // icons in the list view without the vertical-align: middle;
```

Comment:
nitpicking but... should be a `FIXME:` instead of a `TODO` to strongly mark the intent "remove this cr** as soon as possible" :wink: